README.md

```diff
@@ -194,6 +194,24 @@ dense.out_ports.a_out.connect(lif2.in_ports.a_in)
 # Execute process lif1 and all processes connected to it for fixed number of steps
 from lava.magma.core.run_conditions import RunSteps
 from lava.magma.core.run_configs import RunConfig
+
+from lava.magma.core.run_conditions import RunSteps
+from lava.magma.core.run_configs import RunConfig
+
+class SimpleRunConfig(RunConfig):
```

Comment:
Can't we just use a predefined RunConfig rather than making this example more complicated? Just import Loihi1SimConfig
This is not so helpful without explanation. 

I also think the sync_domains is not a mandatory input. For simplicity I would leave that out.

Reply:
`custom_sync_domains` is mandatory because a list is expected but the default is `None`, thus it will through an error if a  list is not provided.

https://github.com/lava-nc/lava/blob/81336e63783edf6b27f2f019797728b208458cb8/src/lava/magma/core/run_configs.py#L75-L84

This is a bug and should be fixed to `[]` in another issue.